packages/twenty-server/src/workspace/workspace-sync-metadata/standard-objects/calendar-channel.object-metadata.ts

```diff
@@ -90,4 +93,20 @@ export class CalendarChannelObjectMetadata extends BaseObjectMetadata {
     icon: 'IconReload',
   })
   syncCursor: string;
+
+  @FieldMetadata({
+    type: FieldMetadataType.RELATION,
+    label: 'Calendar Channel Event Associations',
+    description: 'Calendar Channel Event Associations',
+    icon: 'IconCalendar',
+  })
+  @RelationMetadata({
+    type: RelationMetadataType.ONE_TO_MANY,
+    inverseSideTarget: () => CalendarChannelEventAssociationObjectMetadata,
+    inverseSideFieldKey: 'calendarChannel',
```

Comment:
I'm not sure but I thought this was not needed anymore if the value was the same as the nameSingular of the object